js/main.js

```diff
@@ -172,3 +172,27 @@ button.forEach((btn) => {
   btn.addEventListener('click', Modalopen);
 });
```

Comment:
Please add all unnecessary files and folders like `vscode` to `.gitignore` so that the project directory remains clean and managed.